faq.md

```diff
@@ -81,18 +167,32 @@ Pull requests will be merged by the training team members after at least one oth
+
+We strongly recommend you read and follow [Software Carpentry's](https://software-carpentry.org/) recommendations on [lesson design](https://swcarpentry.github.io/lesson-example/01-design/) and [lesson writing](https://swcarpentry.github.io/instructor-training/19-lessons/) if you plan to add or change some training materials, and also to check the [structure of the training material](#how-the-training-material-is-structured).
 
-We strongly recommend you read and follow Software Carpentry's recommendations on [lesson design](https://swcarpentry.github.io/lesson-example/01-design/) and [lesson writing](https://swcarpentry.github.io/instructor-training/19-lessons/) if you plan to add or change some training materials, and also to check the [structure of the training material](#how-the-training-material-is-structured).
+# Other Questions
 
-## Is there any interesting events in which I could participate?
+## Are there any upcoming events focused on Galaxy Training?
 
-Contribution Fest
-- GCC
-- Every 3 months on the 3rd Friday: check our next event
+Yes.  As of May 2018, these events are on the horizon:
+
+* [CarpentryCon 2018](http://www.carpentrycon.org/), 30 May - 1 June, Dublin, Ireland
+  * Not specifically about Galaxy Training, but an excellent opportunity to gather with other computational science educators. [Bérénice Batut](@bebatut) will present a poster and lightning talk on [Community-Driven Training for Biological Data Analysis with the Galaxy Training Network](https://github.com/carpentries/carpentrycon/blob/master/Sessions/2018-05-30/13-Poster-Session/abstract-berenice-batut.md)
+* [GCCBOSC 2018](https://gccbosc2018.sched.com/), June 25-30, Portland, Oregon, United States
+  * The annual gathering of the Galaxy Community is an opportunity to learn from experienced Galaxy trainers and to contribute to these efforts:
+    * [Bioinformatics Training and Education with the Galaxy Training Network](http://sched.co/Drp9), training session on how to use and contribute to these materials, presented by [Bérénice Batut](@bebatut)
+    * [A fruitful year for the Galaxy Training materials]()http://sched.co/EYF1, conference talk by [Bérénice Batut](@bebatut)
+    * [Panel: Training and Documentation in Bioinformatics](http://sched.co/Dup7)
+    * The *Galaxy documentation, analysis, and training (Galaxy DAT)* track of  [CollaborationFest](https://galaxyproject.org/events/gccbosc2018/collaboration/), June 29 - July 2. Focus on expanding Galaxy community resources like training materials and documentation.
+* Quarterly online training material Contribution Fests: The training community will meet online on the 3rd Friday of every 3rd month to focus on enhancing particular areas of the training material.
+
+Is the above list now out of data (it happens).  See the [Galaxy Community Events Calendar](https://galaxyproject.org/events/) for what coming up right now.
 
 # How is the training material structured?
```

Comment:
We could maybe add here (or somewhere else) the "big" picture: 

![infra_4](https://user-images.githubusercontent.com/1842467/40478422-4b621c30-5f49-11e8-81e2-b29aadc277b1.png)

(or the one from the paper) with a legend

I will also change the training introduction slide deck to describe this "big" picture

Reply:
also the cover-image would be nice